.github/workflows/ci-comment.yml

```diff
@@ -32,10 +32,17 @@ jobs:
             let fs = require('fs');
             fs.writeFileSync(`${process.env.GITHUB_WORKSPACE}/pr_number.zip`, Buffer.from(download.data));
 
-      - name: 'Unzip artifact'
+      - name: Unzip Artifact
         run: unzip pr_number.zip
+      
+      - name: Delete Previous Comment
+        uses: izhangzhihao/delete-comment@master
+        with: 
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          delete_user_name: github-actions[bot]
+          issue_number: ${{ github.event.number }}  # remove comments from the current PR
```

Comment:
Why remove previous comments? I've always preferred keeping everything in a coherent timeline, so you can see the history, but I guess I could be convinced otherwise.

Reply:
It clutters history. Very fast.

If you want to see the history of which checks passed and failed, the checks tab allows you to pull up the run results for a specific commit.